systemtests/tests/python-bareos/test_list_command.py

```diff
@@ -335,8 +384,7 @@ def test_list_media(self):
             **self.director_extra_options,
         )
 
-        director.call("run job=backup-bareos-fd yes")
-        director.call("wait")
+        self.get_backup_jobid(director, "backup-bareos-fd")
```

Comment:
I feel like this should require `level=Full` here when looking at the following lines.